elasticsearch_async/connection.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, host='localhost', port=9200, http_auth=None,
             connector=aiohttp.TCPConnector(
                 loop=self.loop,
                 verify_ssl=verify_certs,
-                conn_timeout=self.timeout,
+                keepalive_timeout=self.timeout,
```

Comment:
`conn_timeout` is no longer there, the docs on `TCPConnector` are a out of date. But you're right we shouldn't use `keepalive_timeout` here, that makes no sense as a replacement. The only other timeout we can use is `ClientSession.read_timeout`, which is cumulative. But that seems to be rounded off to seconds. I'll dig a little deeper soon.